sycl/source/detail/memory_manager.cpp

```diff
@@ -176,27 +175,40 @@ void copyH2D(SYCLMemObjT *SYCLMemObj, char *SrcMem, QueueImplPtr SrcQueue,
   DstAccessRange[0] *= DstElemSize;
   SrcSize[0] *= SrcElemSize;
   DstSize[0] *= DstElemSize;
-
-  cl_command_queue CLQueue = UseExclusiveQueue
```

Comment:
Is there a reason that we remove this and specifically use TgtQueue->getHandleRef() rather than allowing the caller to specify it via a bool? I'd like to keep this as close to the upstream SYCL implementation as possible unless there is good reason